purchases/src/main/kotlin/com/revenuecat/purchases/backup/RevenueCatProactiveBackupAgent.kt

```diff
@@ -0,0 +1,29 @@
+package com.revenuecat.purchases.backup
+
+import android.app.backup.BackupAgentHelper
+import android.app.backup.BackupDataInput
+import android.app.backup.BackupDataOutput
+import android.app.backup.SharedPreferencesBackupHelper
+import android.os.ParcelFileDescriptor
+import com.revenuecat.purchases.common.SharedPreferencesManager
+import com.revenuecat.purchases.common.debugLog
+
+private const val REVENUECAT_PREFS_BACKUP_KEY = "revenuecat_prefs_backup"
+
+class RevenueCatProactiveBackupAgent : BackupAgentHelper() {
```

Comment:
TBH, I'm not sure of the name... Maybe just `RevenueCatBackupAgent` makes more sense? The idea was to add docs to make it a "proactive restores" feature...